src/org/exoplatform/ui/login/tasks/LoginTask.java

```diff
@@ -75,6 +75,12 @@ public Integer doInBackground(String... params) {
   }
 
   @Override
+  protected void onCancelled() {
```

Comment:
The preferred way to override `onCancelled` is 
- override `onCancelled(Result result)`
- do all necessary cleanup
- call `onCancelled()` without param

cf http://developer.android.com/reference/android/os/AsyncTask.html#onCancelled%28Result%29